docs/index.md

```diff
@@ -35,6 +36,22 @@ Additionally, we prioritise the security of the data during its processing and t
 
 ![](_static/structure.png)
 
+### Contributions
+
+Besides our [models](https://www.foundationmodels.dk/models/). The Danish Foundation models project have led to contributions for a series of open-source projects, the following table include some of these contributions:
+
+
+| Project                                                                                                                | Contribution                                                  |
+| ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| **Packages**                                                                                                           |                                                               |
+| [NLPDedup](https://github.com/saattrupdan/NLPDedup)                                                                    | A deduplication library derived from DFM's deduplication code |
```

Comment:
Wouldn't ScandEval make sense here? It literally started since we were training encoder models in DFM and needed a way to evaluate them.

Reply:
I would say so - will also add SEB (essentially I want it to be very clear that DFM also produce a series of other positive downstream effects)